src/pages/BirthDay.jsx

```diff
@@ -1,5 +1,7 @@
-import CakeIcon from "@mui/icons-material/Cake";
 import { styled, Stack, Typography } from "@mui/material";
+import CakeIcon from "@mui/icons-material/Cake";
+import Chip from "@mui/material-next/Chip";
```

Comment:
```suggestion
import { styled, Stack, Typography, Chip } from "@mui/material";
```
기본 mui에도 Chip컴포넌트가 있는데 `@mui/material-next/Chip`의 Chip을 사용하는 이유가 있는지?

Reply:
Chip 디자인이 `@mui/material-next/Chip`의 Chip 나은거 같아서 사용했습니다....ㅋㅋㅋㅋㅋㅋㅋ

Reply:
디자인은 커스텀 할수 있기 때문에 @mui/material-next/ 패키지는 삭제하고 @mui/material로만 진행하도록 하겠습니다!
패키지를 설치하는건 번들 사이즈에 영향이 크기 때문에 꼭 필요하지 않은경우 지양해야하고 신중히 선택해야합니다~